src/components/progress-circle/progress-circle.spec.ts

```diff
@@ -74,6 +74,22 @@ describe('MdProgressCircular', () => {
         done();
       });
   });
+
+  it('should clean up the indeterminate animation when the element is destroyed', (done: () => void) => {
+      builder
+        .overrideTemplate(TestApp, '<md-progress-circle mode="indeterminate" *ngIf="!isHidden"></md-progress-circle>')
+        .createAsync(TestApp)
+        .then((fixture) => {
+          fixture.detectChanges();
+          let progressElement = getChildDebugElement(fixture.debugElement, 'md-progress-circle');
+          expect(progressElement.componentInstance._interdeterminateInterval).toBeTruthy();
```

Comment:
You can't access the private member here; The field either needs to be changed to `/** @internal */` or we need some other proxy for checking that the interval has stopped.